src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -134,7 +134,7 @@ public void completed(AbstractBuild r) {
         if (null != previousBuild) {
             do {
                 previousBuild = previousBuild.getPreviousCompletedBuild();
-            } while ((null != previousBuild && previousBuild.getResult() == Result.ABORTED) || (null != previousBuild && previousBuild.getNumber() == r.getNumber()));
+            } while (null != previousBuild && previousBuild.getResult() == Result.ABORTED);
```

Comment:
why change this?

Reply:
I screwed up my branches when I first forked the repo.  I made the above bug fix in master, made a PR, then made a branch for this feature.  I undid that work in the feature branch and forgot to add it back in.

TL;DR: Whoops